File: BackendAPI/Application/DTOs/Output/OrderKwekerOutput.cs
using Domain.Enums;

namespace Application.DTOs.Output;

public class OrderKwekerOutput
{
    public required Guid Id { get; set; }
    public required DateTimeOffset CreatedAt { get; set; }
    public required OrderStatus Status { get; set; }
    public required DateTimeOffset? ClosedAt { get; set; }
    public required int Quantity { get; set; }
    public required decimal TotalPrice { get; set; }
    public required ProductOutputDto Product { get; set; }
    public required KoperInfoOutputDto KoperInfo { get; set; }
}

File: FrontendWeb/src/declarations/dtos/output/OrderKwekerOutput.ts
// OrderKwekerOutput.ts
import {OrderStatus} from '../../enums/OrderStatus';
import {ProductOutputDto} from './ProductOutputDto';
import {KoperInfoOutputDto} from './KoperInfoOutputDto';

export interface OrderKwekerOutput {
	id: string;
	createdAt: string;
	status: OrderStatus;
	closedAt: string | null;
	quantity: number;
	totalPrice: number;
	product: ProductOutputDto;
	koperInfo: KoperInfoOutputDto;
}
